### ISSUE TITLE
Email field was not returned in the profile parameter

### ISSUE THREAD

**Issue by NoverNobida** [casdoor/casdoor#861]
I use casdoor OIDC for [Outline](https://github.com/outline/outline), the config is 
```
OIDC_AUTH_URI=https://auth.example.com/login/oauth/authorize?client_id=6a0f6fa5fwvre43b19&response_type=code&redirect_uri=https://outline.example.com/auth/oidc.callback&scope=read&state=casdoor
OIDC_TOKEN_URI=https://auth.example.com/api/login/oauth/access_token
OIDC_USERINFO_URI=https://auth.example.com/api/get-account
OIDC_USERNAME_CLAIM=email
OIDC_DISPLAY_NAME=Casdoor
OIDC_SCOPES="openid email profile"
```
But I got the error `An email field was not returned in the profile parameter`. Maybe `OIDC_AUTH_URI` `OIDC_TOKEN_URI` or `OIDC_USERINFO_URI` is not correct? Or the profile that returned did not contain `email`?

**Comment by hsluoyz** [comment]
@Abingcbc

**Comment by Abingcbc** [comment]
@NoverNobida 
```
OIDC_CLIENT_ID=<>
OIDC_CLIENT_SECRET=<>
OIDC_AUTH_URI=http://auth.example.com/login/oauth/authorize
OIDC_TOKEN_URI=http://lauth.example.com/api/login/oauth/access_token
OIDC_USERINFO_URI=http://auth.example.com/api/userinfo
OIDC_USERNAME_CLAIM=user.email
```

**Comment by RyaoChengfeng** [comment]
@NoverNobida Obviously, `OIDC_AUTH_URI` `OIDC_TOKEN_URI`  `OIDC_USERINFO_URI` is correctly configured.

I think the reponse body of `/api/git-account` not fit in oidc. Please try `OIDC_USERNAME_CLAIM=sub` and see if it works so that I can identify the problem.

**Comment by RyaoChengfeng** [comment]
> @NoverNobida
> 
> ```
> OIDC_CLIENT_ID=<>
> OIDC_CLIENT_SECRET=<>
> OIDC_AUTH_URI=http://auth.example.com/login/oauth/authorize
> OIDC_TOKEN_URI=http://lauth.example.com/api/login/oauth/access_token
> OIDC_USERINFO_URI=http://auth.example.com/api/userinfo
> OIDC_USERNAME_CLAIM=user.email
> ```

@Abingcbc Why use `user.email` instead of `email` ? I think it's just that `OIDC_USERINFO_URI` is not configured correctly.

**Comment by Abingcbc** [comment]
@NoverNobida Yes, you are right
